### ISSUE TITLE
Get coordinates for waypoints when inputting string

### ISSUE THREAD

**Issue by Henriquelay** [bramus/react-native-maps-directions#152]
I want to draw markers on waypoints and have the library's polyline go through them, similar to Google Maps, but I can't seem to get the inputted waypoints' coordinates for creating the markers, or better explained: I can't differentiate between a route leg/arc and a waypoint coordinate.

Before, I was doing the API call on backend and using the full api return json, and there is a field called `geocoded_waypoints` which is perfect for this operation

Any way to access that field on this library or am I missing something ?

**Comment by Henriquelay** [comment]
Sorry, I didn't actually used that field (but I had plans for it in the future, still have interest in it), because Marker don't actually take place IDs.

I actually did something like `routes[0].legs.map((elem, index) => (<Marker coordinate={elem.start_adress} key=index ... >));`

Still, any way to get the full API response, or I will have to modify the library to my needs _(perhaps later make a PR)_ ?

**Comment by Henriquelay** [comment]
I got around this by adding the field `json` to the return object, containing the full json of the request.

I am not sure if I should create a PR, this wasn't an ellegant sollution and I'm not sure if this meets with this libraries' design principles.

All I did was add `acc.json = json,` at line 202, getting it into `result` variable then `onReady` would call with that variable with the full, untreated json for me to access.
